sdk/typescript/src/utils/verify.ts

```diff
@@ -19,7 +19,7 @@ export async function verifyMessage(
 	serializedSignature: SerializedSignature,
 	scope: IntentScope,
 ) {
-	const signature = fromSerializedSignature(serializedSignature);
+	const signature = fromSerializedSignature(serializedSignature)[0];
```

Comment:
Should this throw if there's more than 1?

Reply:
revised!